ui/helpers/job.js

```diff
@@ -85,6 +85,29 @@ export const isUnclassifiedFailure = function isUnclassifiedFailure(job) {
   return thFailureResults.includes(job.result) && !isClassified(job);
 };
 
+export const isIntermittent = function isIntermittent(jobToEvaluate, group) {
+  if (jobToEvaluate.result !== 'testfailed') {
+    return false;
+  }
+
+  console.log(jobToEvaluate);
```

Comment:
if we leave this in, can we add context around this?  I don't know why we will see random job objects in the console- I would rather see something like:
console.log('consider job to be isIntermittent: ' + jobToEvaluate);